pkg/storage/buckets/store.go

```diff
@@ -166,27 +182,82 @@ func (b *BucketStore) List(ctx context.Context, startAfter, endBefore string, li
 }
 
 // convertMeta converts stream metadata to object metadata
-func convertMeta(m objects.Meta) (Meta, error) {
-	var cipher storj.Cipher
-
-	pathEncType := m.UserDefined["path-enc-type"]
+func convertMeta(m objects.Meta) (out Meta, err error) {
+	out.Created = m.Modified
+	// backwards compatibility for old buckets
+	out.PathEncryptionType = storj.AESGCM
+	out.EncryptionScheme.Cipher = storj.Invalid
```

Comment:
I doubt that defaulting the `storj.Invalid` helps backward compatibiltiy. `storj.EncAESGCM` is a much better shot.

Reply:
Invalid makes more sense as a fall-back bucket default, because we can tell whether or not there actually is a bucket default set.